src/VisualStudio/Core/Def/Implementation/Remote/ServiceHubRemoteHostClient.cs

```diff
@@ -31,6 +31,7 @@ private enum GlobalNotificationState
 
         private readonly JsonRpc _rpc;
```

Comment:
❓ Why is this not using `JsonRpcEx`, which already handles this case?

Reply:
JsonRpcEx is an abstract type. and it was not designed for this connection.

this connection is the master control connection. but sure I can create a concrete type for this case, but just looking through code, that require some refactoring, so I guess that is out of this PR's scope for now. but sure, I can open an issue for that.

https://github.com/dotnet/roslyn/issues/34555